### DISCUSSION TITLE
Charging LiFePo4 Batteries below 0°C

### DISCUSSION THREAD

**Discussion by schlimmchen** [hoylabs/OpenDTU-OnBattery#687] [Q&A]
My battery pack consists of 16 LiFePo4 cells from EVE (hopefully...) with a capacity of 230Ah. The datasheet I was supplied with says that the cells can be charged at 0°C and warmer. The standard continous charge current is 0.5C, and the max continuous charge current is 1C. Pulse current is 2C. Discharging is allowed as per datasheet down to -30°C, so that's not a limiting factor at all (for me).

So I read it like this: I may safely charge the battery with 230A at 0°C. If I want to be kind to the cells, I only use up to 115A.

I actually charge (and discharge) with a maximum current of 40A (conservatively, a technical hard limit), which is less than 0.2C.

I cannot understand that charging below 0°C at any rate should always be an issue and avoided at all cost.

What I think I know: With colder temperatures, the electrolyte changes its properties. If charged with a high current, the electro-chemical processes in the battery are not working as designed any more, and the Lithium starts to build up as metallic lithium on the... anode (?), leading to permanent damage (capacity reduction or even shorts).

My feeling is that charging at low rates even at lower temperatures is fine, as long as it is slow enough such that the chemical processes in the battery can happen as designed.

Given that a LiFePo4 battery still retains 50% capacity down to -10°C (conservatively), it may be safe to charge these cells at -10°C with up to 57.5A (C is 115Ah with the assumed 50% reduction, and 0.5C is a safe charging/discharging current, so (115/2) Amperes).

Does this makes sense?

It would free me from thinking about the next winter. I could just use the battery as normal. The cells will never be below -10°C in my climate/setup, and I can at most pump in or discharge 40A, which I then consider safe even at that temperature.

Who can chime in on this? @hoschiking Maybe you can give feedback?

**Comment by spcqike** [comment]
not sure if this applies to all manufacturers but

> When charging lithium iron phosphate batteries below 0°C (32°F), the charge current must be reduced to 0.1C and below -10°C (14°F) it must be reduced to 0.05C. 

https://www.relionbattery.com/knowledge/how-do-lifepo4-batteries-perform-in-cold-temperatures

on the other hand

> Many battery users are unaware that consumer-grade lithium-ion batteries cannot be charged below 0°C (32°F). Although the pack appears to be charging normally, plating of metallic lithium occurs on the anode during a sub-freezing charge that leads to a permanent degradation in performance and safety.

https://batteryuniversity.com/article/bu-410-charging-at-high-and-low-temperatures

Christian ("Der Kanal") tested this with an 12V 100Ah battery
https://www.youtube.com/watch?v=_a19dazfND8

if i remember correctly, he charged and dischared with 10A, so 0.1C, and the result wasn't to great for the battery health.
spoiler:
![grafik](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/50098985/40d12561-7801-49c0-8287-7def90cf123c)

**Comment by hoschiking** [comment]
Ich antworte mal auf deutsch, geht einfacher.
Generell ist es richtig, Laden unter 0°C ist als kritisch zusehen, da die Degradation der Zellen exponentiell zunimmt.
Alles was du beschreibst, ist in etwa korrekt und bei DIY-Projekten durchsaus "umschiffbar".
Nur die Hersteller sind da anders unterwegs. Wie willst du das Thema einem Endkunden verkaufen?
Jahrelange Garantie bieten usw... Daher wird jeglicher Betrieb unter 0°C untersagt.
Ich persönlich würde den Speicher unter 5°C nicht laden, die Kapazität nimmt dann doch mit der Zeit ab,
damit hast du am Ende nichts gewonnen. Was ich empfehlen würde, ist immer eine Heizung einzubauen. So aufwändig ist es nicht.
Den Speicher in Steinwolle isolieren, je dicker desto besser. Allein damit würdest du in 90% der kalten Jahreszeit
gar keine Heizung benötigen, die Eigenerwärmung würde da ausreichen. Dein JK BMS hat ja 2 Temperatursensoren,
damit kannst du prima die Temperatur überwachen und bei Unterschreitung die Heizung anmachen. Da reichen schon
2 Heizmatten je 20W schon aus. Ja, es kostet Energie, den Speicher bei niedrigen Temperaturen zu laden kostet es aber auch,
da der Wirkungsgrad unter 5°C rapide abnimmt. Hinzu kommt, dass es wenn es draussen firert, in der Regel die Sonne scheint und da hat man auch im Winter genug Leistung zur Verfügung um die Heizung zu betreiben.
Für PV-Speicher fü rden Outdoor Betrieb würde ich immer die Zellen von Winston empfehlen.
Die sind Yttrium dotiert und bis -40°C zugelassen (die wurden ursprünglich für militärische Zwecke entwickelt).
Etwas teuerer, aber robuster. Habe mit den Zellen einen 1kWh Akku für meinen Camper gebaut, funktioniert seit 6 Jahren blendend.
Meine Meinung, um eine Heizung kommst du nicht herum. Es ist auf die Dauer günstiger.

**Comment by schlimmchen** [comment]
Danke für eure Rückmeldung.

Du, @hoschiking, scheinst ja durchaus einzuräumen, dass es auch unter 0°C ginge, aber gleichzeitig sagst du, dass du selbst nicht unter 5°C laden würdest.

Naja, dann muss ich das wohl noch angehen. Die Batterie im Winter abzuklemmen passt mir jedenfalls auch nicht, weil der Inverter logischerweise nicht gut mit dem charge controller zusammenarbeitet ohne eine stützende Kapazität.

@hoschiking Hast du einen Link zu so einer Heizplatte? Ich finde eigentlich nur Heizmatten für Reptilien.

**Reply by hoschiking** [reply]
In meiner Antwort sehe ich keinen Widerspruch ;-)
Man kann es probieren, dann siehst du erst in einigen Monaten / Jahren ob es deinem Speicher geschadet hat.
Da man es vorab schlecht testen kann ohne den Speicher zu opfern, würde ich mich an die Herstellerdaten halten.
Mit dein Heizmatten hab ich keine Erfahrungen, es spricht aber nichts dagegen welche zu nehmen, die für Reotilien gedacht sind.
Mein Chef ist auf Reisen, den könnte ich fragen wenn er zurück ist. Er hat da Erfahrungen damit.
Was ich auf die Schnelle im Netz gefunden habe, ist das was du bereits gefunden hast. Das z.B.
https://www.reptilienkosmos.de/terraristik-shop/technik-und-klima/heizung/dragon-flat-mat-heizmatte_987_4379
Sowas scheint bestens geeignet zu sein. Direkt mit 230V betreiben finde ich sinnvoll, da bist du nicht vom Ladezustand des Speichers abhängig.
